### DISCUSSION TITLE
Running on a Surface Pro 7

### DISCUSSION THREAD

**Discussion by jnlmccall** [kg4vdk/arcos-linux-modules#18] [General Discussion]
Runs nicely, so far, on my older Surface Pro 7. Waiting on digirig hardware for further testing, I'll be out in the field 8/22 - 25, I'll let you know how it goes.

![20240812_185949908_iOS](https://github.com/user-attachments/assets/490b37e2-e8e4-40f9-af14-6bc50fb10a94)

**Comment by kg4vdk** [comment]
I look forward to the After Action Report!

**Comment by brandonb927** [comment]
@jnlmccall Can you report back on your usage here? I can't say I haven't been interested in picking up a used Surface Tablet to give it the arcOS treatment, but I don't want to shell out $$$ until I know its at least stable in the field.

**Reply by jnlmccall** [reply]
It's been just fine. I found I have to run the USB drive directly on the tablet and run everything else on a hub plugged into the USB-C port. Really my only complaint is, as a regular Linux user, I used to being able to apt update/upgrade to keep stuff up to date and secure. This distro doesn't let you do that because of the locked OS.
I connect it to my AT-778U and send emails via Winlink and APRS when camping, an RTL-SDR for general scanning and ADS-B monitoring, and a GPS receiver. I would love to see a SSTV decoder added as I love do the ISS SSTV events, and a weather fax decoder. I know most of the satellite based stuff for Linux is not maintained anymore but gpredict and WxToIMG I think would make this a complete package for me.

**Reply by brandonb927** [reply]
Ok this is good info! I have two suggestions for you: 
- have you looked at installing the OS onto the internal storage? Check the last bit of the "Golf" release video https://youtu.be/xe6mucTuQF4?si=8_-7jSBz4vplAXij&t=1554
- I have a working [QSSTV](https://github.com/ON4QZ/QSSTV) module that I will be submitting to the Community modules sometime soon after more testing, if @kg4vdk wants to add it

**Reply by jnlmccall** [reply]
I have run various distros (Ubuntu, Fedora, Kali) directly on the Surface storage before and they worked just fine. I was running this from the USB to try it out, the reason for which USB ports is because of the power the surface pushes to the USB-C port isn't enough for the drive, GPS dongle, and the Digirig to all run from the USB hub. I have not tried a powered hub, I'm trying to keep this portable. I have not tried installing this directly to storage yet.

Awesome you are working on the QSSTV module, I look forward to that!

**Reply by brandonb927** [reply]
@jnlmccall just wanted to chime in here and mention that arcOS "Crater Lake" has QSSTV baked in! I never could get it working right, I almost did but I ran into a few bugs (that I don't quite remember the details on) that I'm hoping @kg4vdk worked out in this release.
